src/zonetree/answer.rs

```diff
@@ -87,6 +95,11 @@ impl Answer {
         self.authority = Some(authority)
     }
 
+    /// Marks the response authoritative for the QNAME.
```

Comment:
Documentation nit-pickery: The method marks _whether_ the response (also, this should probably be “answer”) is authoritative – i.e., the text should reflect the presence of the boolean argument.

Reply:
Ah, it didn't have the argument when I first wrote the documentation! Good catch.

Reply:
Done. Please check.